sandy-city-utah/sandy-city-logger
# feat(options): add logging options and initial cleanup

### What Changed?
  - add logic to refresh log file if process is started another day
  - add ability to provide log options to the log functions
  - include console suppression configuration

### Issues Addressed:
  - P21312-17

### Issue Testing:
  - Manual Developer Testing

### Test Results:
  1. Run one of the dependent processes
  1. See that the log file is generated as expected
  1. Either kill the process or wait for it to complete
  1. Either wait for the date to change or change your own date/time settings
  1. Run the process again
  1. See that the old log file is moved and the new one is created as expected
  1. Running the API, see that there are not duplicate log messages when the application has started up (suppressed console)
